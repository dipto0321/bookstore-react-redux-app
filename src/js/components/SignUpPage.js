import React from 'react';
import UserForm from './UserForm';

const SignUpPage = () => (
  <div>
    <h2>Create Account</h2>
    <UserForm />
  </div>
);

export default SignUpPage;
