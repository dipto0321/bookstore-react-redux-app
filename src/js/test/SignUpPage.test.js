import React from 'react';
import { shallow } from 'enzyme';
import SignUpPage from '../components/SignUpPage';

test('SignUpPage component renders correctly', () => {
  const wrapper = shallow(<SignUpPage />);
  expect(wrapper).toMatchSnapshot();
});
