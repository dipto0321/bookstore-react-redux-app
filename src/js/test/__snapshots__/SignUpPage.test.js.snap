// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUpPage component renders correctly 1`] = `
<div>
  <h2>
    Create Account
  </h2>
  <ConnectFunction />
</div>
`;
